CRM/Contribute/BAO/ContributionPage.php

```diff
@@ -742,11 +742,15 @@ public static function getSectionInfo($contribPageIds = []) {
       'membership',
       'custom',
       'thankyou',
-      'friend',
       'pcp',
       'widget',
       'premium',
     ];
+
+    if (function_exists('tellafriend_civicrm_config')) {
```

Comment:
```suggestion
    if (\Civi\Api4\Extension::get(FALSE)->addWhere('file', '=', 'tellafriend')->addWhere('status', '=', 'installed')->execute()->selectRowCount()->count()) {
```

It seems a bit more legit to use Api4. Maybe there's a shorter way to write this?

Reply:
Zooming out, I think the question to ask is "how should extensions modify core", to which the answer is "implement hooks!" I haven't looked too close but for this specific spot maybe hook_tabset, or buildForm, can do it. But then zooming sideways, is that amount of refactoring something you want to take on in this PR.

Reply:
For now I think this is fine. Further refactoring is much easier once this PR is merged. Feel free to follow up with this change or an alternative using hooks.